### PR TITLE
cluster tasks replaced with ecosystem tasks and cluster resolver

### PR DESCRIPTION
**Pull request by ppitonak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,16 +167,17 @@ apply-manifests     10 seconds ago
...
-buildah-v0-14-3                          1 day ago
-git-clone                                1 day ago
-s2i-php                                  1 day ago
-tkn                                      1 day ago
+$ tkn tasks ls -n openshift-pipelines
+NAME                      DESCRIPTION              AGE
+buildah                   Buildah task builds...   1 day ago
+git-cli                   This task can be us...   1 day ago
+git-clone                 This object represe...   1 day ago
+hello                                              1 day ago
```
Comment (savitaashture) [inline README.md]: what is this `hello` task  ??

**Review by savitaashture — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ppitonak) [reply README.md]: My bad, I was testing something on my cluster. Fixed

Comment (savitaashture) [reply README.md]: no worries 
Thank you

**Comment by ppitonak** [comment]
Please do not forget to cherry-pick to 1.17 branch

**Pull request merged**